RecoPixelVertexing/PixelVertexFinding/test/VertexFinder_t.h

```diff
@@ -127,48 +140,56 @@ int main() {
 
   float eps = 0.1f;
   std::array<float, 3> par{{eps, 0.01f, 9.0f}};
-  for (int nav = 30; nav < 80; nav += 20) {
+  for (int nav = 30; nav < 100; nav += 20) {
     ClusterGenerator gen(nav, 10);
 
-    for (int i = 8; i < 20; ++i) {
-      auto kk = i / 4;  // M param
+    for (int iii = 8; iii < 20; ++iii) {
```

Comment:
why change index from `i` to `iii` ?

Reply:
because I used ```i``` in some internal loops.
I can be more expressive in those external loops, yes.